chore(*): update-overview-topics (#1034)
IgniteUI/igniteui-xplat-docs: doc/en/components/grids/_shared/column-pinning.md

# {Platform} {ComponentTitle} Column Pinning

> The {ProductName} Column Pinning feature in {Platform} {ComponentTitle} enables developers to benefit from the  that lets them lock specific columns in a desired order, ensuring visibility all the time even when users scroll horizontally through the `Grid`. There’s an integrated UI for Column Pinning, accessible via the {Platform} {ComponentTitle} toolbar. Additionally, developers have the flexibility to build a custom user interface and change the pin state of the columns.

Reviewer: ```suggestion
The {ProductName} Column Pinning feature in {Platform} {ComponentTitle} enables developers to lock specific columns in a desired order, ensuring visibility all the time even when users scroll horizontally through the `Grid`. There’s an integrated UI for Column Pinning, accessible via the {Platform} {ComponentTitle} toolbar. Additionally, developers have the flexibility to build a custom user interface which changes the pin state of the columns.
```
